pkg/stdio/stdio_internal_test.go

```diff
@@ -0,0 +1,284 @@
+package stdio
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTerminalStdIO_Banner(t *testing.T) {
+	t.Parallel()
+
+	tmp, err := os.MkdirTemp("", "")
+	require.NoError(t, err)
+	defer os.RemoveAll(tmp)
+
+	stdin, err := os.Create(filepath.Join(tmp, "stdin"))
+	require.NoError(t, err)
+	defer stdin.Close()
```

Comment:
Non-blocking: is it worth checking the return from the Close() calls?

Reply:
I don't typically check them (especially in test code), I figure if `Close()` fails then some other catastrophic thing must have happened that will show in the test output